test/e2e/framework/volume/fixtures.go

```diff
@@ -511,8 +511,9 @@ func TestVolumeClientSlow(f *framework.Framework, config TestConfig, fsGroup *in
 }
 
 func testVolumeClient(f *framework.Framework, config TestConfig, fsGroup *int64, fsType string, tests []Test, slow bool) {
+	privileged := true
```

Comment:
@Elbehery, thank you for working on this.  

I have a question for you. Although this fixes the issue,  could you check if you can set `privileged = true` only when the HostPath plugin is being used? Perhaps we could add a flag for that in the plugin definition.

The idea is to avoid always going privileged. Thanks!